astropy_helpers/openmp_helpers.py

```diff
@@ -39,6 +41,34 @@
 }
 """
 
+def _split_option_from_var(option, var, delim=' '):
```

Comment:
This function needs a docstring for clarity (astropy-helpers code has typically been under-documented in the past, causing problems down the road, so we need to make sure all new code is easy to understand/read). Also please include some more comments below in the function to explain a bit more what is going on.

Reply:
I'm going to slightly refac this and rename it, sorry.